src/WinXP/utils.js

```diff
@@ -6,3 +8,7 @@ export function getLocalStorage(name) {
   const item = localStorage.getItem(name);
   return JSON.parse(item);
 }
+
+export const debouncedFunc = debounce(callback => {
+  callback();
+}, 1000);
```

Comment:
As discussed, let's try using `throttle` with something like `10000` so this happens less often